packages/generators/generators/lib/plops/api.js

```diff
@@ -45,7 +46,8 @@ module.exports = plop => {
       },
     ],
     actions(answers) {
-      const filePath = answers.isPluginApi && answers.plugin ? 'plugins/{{plugin}}/server' : 'api/{{id}}';
+      const answerDestination = answers.isPluginApi && answers.plugin ? 'plugin' : 'api';
+      const filePath = getFilePath(answerDestination);
```

Comment:
`getFilePath` isn't compatible with this use case.

The API generator `filePath` wil either be `plugins/ {{ plugin }}/server` or `api/{{ id }}`. But with you changes it will be `plugins/ {{ plugin }}/server` or `api/{{ api }}`.

I would revert this change for simplicity and avoiding adding more code than necessary.